packages/ui-components/src/components/toggle-button/toggle-button.tsx

```diff
@@ -65,24 +67,25 @@ export class KvRadioButton implements IRadioButton, IRadioButtonEvents {
 			<Host>
 				<a
 					class={{
-						'radio-button': true,
-						'radio-button--checked': this.checked,
-						'radio-button--disabled': this.disabled,
-						'radio-button--only-icon': hasIcon && !hasLabel
+						'toggle-button': true,
+						'toggle-button--checked': this.checked,
+						'toggle-button--disabled': this.disabled,
+						'toggle-button--only-icon': hasIcon && !hasLabel
 					}}
-					part="radio-button"
+					part="toggle-button"
 					onClick={this.onClick}
 					download={this.download}
 					href={this.href}
 					target={this.target}
 				>
+					{this.withRadio && <kv-radio checked={this.checked} />}
```

Comment:
This new definition increases the toggle height when enabled:

<img width="849" alt="Screenshot 2023-03-20 at 22 06 20" src="https://user-images.githubusercontent.com/3854939/226477407-d6213fe1-efea-4dd0-abdf-27dbeaad6c3d.png">

Reply:
Now it should be fixed :pray: